lib/builder/builder.js

```diff
@@ -622,6 +623,15 @@ export default class Builder {
     this.customFilesWatcher = chokidar
       .watch(customPatterns, options)
       .on('change', refreshFiles)
+
+    // Watch for nuxt.config.js and user-defined changes
+    const nuxtConfigPatterns = _.concat(
+      this.options.watch,
+      path.join(this.options.rootDir, 'nuxt.config.js')
+    )
+    this.nuxtConfigWatcher = chokidar
+      .watch(nuxtConfigPatterns, options)
+      .on('change', refreshFiles)
```

Comment:
If the purpose is to restart the server, can `refreshFiles` work here ?

Reply:
Woops, good watch, no we need to sent an event from nuxt internal that `nuxt-dev` will listen to kill Nuxt and restart it